src/client/Client.cc

```diff
@@ -11234,7 +11234,7 @@ int Client::write(int fd, const char *buf, loff_t size, loff_t offset)
 #endif
   /* We can't return bytes written larger than INT_MAX, clamp size to that */
   size = std::min(size, (loff_t)INT_MAX);
-  int r = _write(fh, offset, size, buf, NULL, false);
+  int r = _write(fh, offset, size, buf);
```

Comment:
Is this change intentional? `Client::write` isn't vectored write path.

Reply:
Just simplifying all the calls since new defaults of NULL and false were added, so updated all the callers.